Legion/DarkheartThicket/ShadeOfXavius.lua

```diff
@@ -52,6 +54,12 @@ function mod:GrowingParanoiaRemoved(args)
 	self:PrimaryIcon(args.spellId)
 end
 
+function mod:FeedOnTheWeakApplied(args)
+	if self:Me(args.destGUID) or self:Healer() then
+		self:TargetMessage(args.spellId, args.destName, "Urgent", "Alert", nil, nil, true)

```

Comment:
Please change "Urgent" and "Alert" so some other color and sound, it's being used in the module already.

Reply:
Done.